include/blas_meta.h

```diff
@@ -69,6 +69,18 @@ enum class transpose_type : char {
   Conjugate = 'c'
 };
 
+/**
+ * @enum Uplo
+ * @brief Specifies whether the lower or upper triangle needs to be accessed.
+ */
+enum class uplo_type : char { Upper = 'u', Lower = 'l' };
```

Comment:
We have these types used on different functions (TRSM,GER, etc). Maybe not as part of this PR, but if you are planning to use BLAS meta for this, it would be good to align those functions as well.